README.md

```diff
@@ -3,6 +3,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/publiclab/plotsbot/badge.svg?branch=master)](https://coveralls.io/github/publiclab/plotsbot?branch=master)
 [![license](https://img.shields.io/badge/license-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com/)
+[![Dependencies](https://img.shields.io/versioneye/d/ruby/rails.svg)](https://github.com/publiclab/plotsbot#dependencies)
```

Comment:
Hey @prabeous73 , since this is a [node.js](https://nodejs.org/en/) based code, all the dependencies are listed in the `package.json` file. You can have a look at [this](https://www.tutorialspoint.com/nodejs/nodejs_npm.htm) to understand the structure. 

The badges are generated by dependency analyzers (**bitHound** and **David**) using information from this `package.json` file. 

For the badge, you can use either this -
* [**David**](https://david-dm.org/) : [![David Dependencies](https://david-dm.org/publiclab/plotsbot.svg)](https://david-dm.org/publiclab/plotsbot) or 
* [**bitHound**](https://www.bithound.io/) : [![bitHound Dependencies](https://www.bithound.io/github/bassdeveloper/plotsbot/badges/dependencies.svg)](https://www.bithound.io/github/bassdeveloper/plotsbot/master/dependencies/npm) 

and change this line accordingly with the correct URL for both the badge image and the repo dependency details.

NOTE : This would require a URL (sign-in) for **bitHound** so you can go ahead with **David**.